### ISSUE TITLE
KeyError 189 for AZERTY keyboards on _send_event - only for key (-)

### ISSUE THREAD

**Issue by wolfeman95** [boppreh/keyboard#131]
Hey guys, I am making a demo player and trying to make it work with AZERTY keyboards as well as QWERTY.  Basically I made an AZERTY_num_dict for pressing numbers on azerty keyboards, which looks like this:

```Python
azerty_num_dict = {".": "shift+;", "0": "shift+à", "1": "shift+&", "2": "shift+é",
                   "3": "shift+\"", "4": "shift+'", "5": "shift+(", "6": "shift+-",
                   "7": "shift+è", "8": "shift+_", "9": "shift+ç"}
```

So, if the user is using an azerty keyboard- and numbers are needed to be pressed (combo of shift + x on azerty) - they are pressed in this way: 
```python
selection = "Breaking_Dawn_16Feb2018_041619_0markers.rep"

keyboard.write("play ")
for any in str(selection[:-4]):
    if any.isdigit() or any is ".":
        try:
             keyboard.press_and_release(self.azerty_num_dict[any])
        except:
             traceback.print_exc()
    else:
        keyboard.press_and_release(any)
```
My buddy is testing this on an AZERTY keyboard, and every key is working except the number 6.  Here is what he is getting when he tries to play the above selection: 

![demeazerty](https://user-images.githubusercontent.com/24723883/36643302-8a35314e-1a17-11e8-9d3d-714ab1ca5eba.png)

As you can see - only the 6 's are omitted.  And the error:

![keyerror](https://user-images.githubusercontent.com/24723883/36643355-01261df4-1a18-11e8-9209-85c0ed4b83c4.png)

So basically: users with AZERTY keyboards get a keyerror when trying to press and release 6 using: ```keyboard.press_and_release("shift+-")```
The error is pertaining to the minus key (-).  Not sure why the keyerror is 189.... the chr() for minus key is 45.  

I reproduced the error on my end by changing my keyboard default language to French (France) and simply trying `keyboard.press_and_release("-")`

![fqilg](https://user-images.githubusercontent.com/24723883/36643536-9efee676-1a1a-11e8-966b-118f03564836.png)

Not sure if this is a possible bug with keyboard package?  Any help would be greatly appreciated I am at a loss for where to go from here.

**Comment by boppreh** [comment]
Looks like a bug, yes. Note that it's looking for the key `\``, not minus. I'll try to reproduce it, give me a moment.

Just to be clear, the line `keyboard.press_and_release("-")` is sufficient to trigger the bug on Windows + French keyboard, correct?

**Comment by wolfeman95** [comment]
> Looks like a bug, yes. Note that it's looking for the key ```, not minus. I'll try to reproduce it, give me a moment.
- no, I caught that ValueError previously in the code in a try/except ValueError- to determine if the user is on AZERTY (as they cannot simply press_and_release("`") on azerty but must use shift+x so it will throw.  

Yes it is sufficient to throw error on Windows + French keyboard layout here is what my settings are:

![keyboqrd](https://user-images.githubusercontent.com/24723883/36643662-e8ccf8fe-1a1c-11e8-843b-0bd2369b2de8.png)

**Comment by boppreh** [comment]
Aha, found the problem. Good news, it has already been fixed a while ago, it just hasn't been included in a release yet. On my own tests v0.11.0 has the bug, but the master branch works as expected. Could you give the master branch a try?

**Comment by wolfeman95** [comment]
YES! Working - thanks :+1:

**Comment by boppreh** [comment]
Awesome, thank you.
